code/online_challenges/src/project_euler/problem_020/Problem_020.java

```diff
@@ -0,0 +1,25 @@
+import java.math.BigInteger;
+
+public class Problem_020 {
+	public static BigInteger factorial(BigInteger number) {
+		if (number.equals(BigInteger.ZERO))
+			return BigInteger.ONE;
+		else
```

Comment:
Actually you can get rid of the else statement entirely and just use the return. Ternary is an option here, but that may be too verbose.